src/components/ManageAccountsDrawer/components/ImportAccounts.tsx

```diff
@@ -45,18 +49,28 @@ type TableRowProps = {
   accountId: AccountId
   accountNumber: number
   asset: Asset
-  toggleAccountId: (accountId: AccountId) => void
+  onAccountIdActiveChange: (accountId: AccountId, isActive: boolean) => void
 }
 
 const disabledProp = { opacity: 0.5, cursor: 'not-allowed', userSelect: 'none' }
 
 const TableRow = forwardRef<TableRowProps, 'div'>(
-  ({ asset, accountId, accountNumber, toggleAccountId }, ref) => {
+  ({ asset, accountId, accountNumber, onAccountIdActiveChange }, ref) => {
     const translate = useTranslate()
-    const handleChange = useCallback(() => toggleAccountId(accountId), [accountId, toggleAccountId])
     const accountLabel = useMemo(() => accountIdToLabel(accountId), [accountId])
     const balanceFilter = useMemo(() => ({ assetId: asset.assetId, accountId }), [asset, accountId])
+    const isAccountEnabledFilter = useMemo(() => ({ accountId }), [accountId])
+    const isAccountEnabledInRedux = useAppSelector(state =>
+      selectIsAccountIdEnabled(state, isAccountEnabledFilter),
+    )
+
+    const [isAccountActive, toggleIsAccountActive] = useToggle(isAccountEnabledInRedux)
+
+    useEffect(() => {
+      onAccountIdActiveChange(accountId, isAccountActive)
+    }, [accountId, isAccountActive, isAccountEnabledInRedux, onAccountIdActiveChange])
```

Comment:
If we absolutely need to keep this vs. the previous `onToggle`, then this could be moved to l.87 with a `(accountId) => onAccountIdActiveChange(accountId, isAccountActive)` lambda` instead

Reply:
good point, ill fix this in a follow-up as it'll introduce conflicts with the next PR